src/status_im2/subs/shell.cljs

```diff
@@ -103,11 +103,18 @@
                  #(re-frame/dispatch [:chat/navigate-to-chat channel-id])
                  100))}))
 
+(def memo-shell-cards (atom nil))
+
 (re-frame/reg-sub
  :shell/sorted-switcher-cards
  :<- [:shell/switcher-cards]
- (fn [stacks]
-   (sort-by :clock > (map val stacks))))
+ :<- [:view-id]
+ (fn [[stacks view-id]]
+   (if (= view-id :shell)
+     (let [sorted-shell-cards (sort-by :clock > (map val stacks))]
```

Comment:
@Parveshdhull can we do it in event handler on fetching cards instead of using atom? Or on handler which changes those stacks or something. If we need some pre calculated data I suppose it should happen in event handler.

Reply:
Yes, it makes sense. will change it